src/ocean/io/select/client/FileSystemEvent.d

```diff
@@ -101,11 +167,23 @@ class FileSystemEvent : ISelectClient
 
     ***************************************************************************/
 
+    // TODO: this is deprecated. Remove in the next major
     private Handler handler;
 
 
     /***************************************************************************
 
+        Event handler delegate, specified in the constructor and called whenever
+        a watched file system event fires, passing the SmartUnion describing the
+        event.
+
+    ***************************************************************************/
+
+    private HandlerEx handler_ex;
```

Comment:
Just a thought: to avoid `_ex` / `Ex`, you could call the new thing a "notifier" rather than a handler.